app/src/services/storage.js

```diff
@@ -168,6 +232,24 @@ const objectStorageService = {
     return getSignedUrl(this._s3Client, command, { expiresIn });
   },
 
+  /**
+   * @function putBucketEncryption
+   */
+  async putBucketEncryption() {
+    const params = {
+      Bucket: bucket,
+      ServerSideEncryptionConfiguration: {
+        Rules: [{
+          ApplyServerSideEncryptionByDefault: {
+            SSEAlgorithm: 'AES256'
+          }
+        }]
+      }
+    };
+
+    return await this._s3Client.send(new PutBucketEncryptionCommand(params));
```

Comment:
We should not be awaiting and force unwrapping this command as the storage service by convention always returns a promise.